### PR TITLE
odd questions answered by Anna Hrychaniuk(London10)

### PR DESCRIPTION
**Pull request by AnnaFYZ**

### CODE REVIEW

File: questions-js1/1.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Q: How do you write "Hello World" to the console?
 
-A: {YOUR ANSWER HERE}
+A: console.log("Hello World");
```
Comment (PaulinaWywrot) [inline questions-js1/1.md:3]: great job

File: questions-js1/3.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Q: What is the difference between `let` and `const`?
 
-A: {YOUR ANSWER HERE}
+A: let can be changed after, const can't be change
```
Comment (PaulinaWywrot) [inline questions-js1/3.md:5]: well done

File: questions-js1/5.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Q: What is the difference between an expression and a statement?
 
-A: {YOUR ANSWER HERE}
+A: expression is a part of code and statement is like a finished function
```
Comment (PaulinaWywrot) [inline questions-js1/5.md:3]: good job

File: questions-js1/7.md [hunk 4]
```diff
@@ -5,6 +5,6 @@ Ctr + Shift + V (Windows/Linux) or CMD + Shift + V (Mac)
 
 In this task, you'll need to look at some code in `7.js` and, firstly, predict what it will do, **without executing it**.
 
-{YOUR PREDICTION HERE}
+will show nothing as there is no command for printing on the screen
```
Comment (PaulinaWywrot) [inline questions-js1/7.md:8]: well done

File: questions-js1/3.md (hunk 2 above)
Comment (AnnaFYZ) [reply questions-js1/3.md:5]: thank you

**Pull request closed without merging**